blocks/subscriptions-block/features/checkout/default.jsx

```diff
@@ -64,47 +59,39 @@ const Checkout = ({ customFields }) => {
 		};
 	}, [Identity, loggedIn]);
 
-	const logoutCallback = () => {
-		Identity.logout().then(() => {
-			setUser(false);
-		});
-	};
-
-	const setUserInfoAndShowPaymentScreen = async (userInfo) => {
-		const { firstName, lastName } = userInfo;
-		setUserInfo(userInfo);
-		if (user) {
-			Identity.updateUserProfile({ firstName, lastName });
+	useEffect(() => {
+		if(!isFetching) {
+			if (loggedIn) {
+				setIsOpen(state => ({...state, account: false, billingAddress: true}))
+				setIsComplete(state => ({...state, account: true}))
+				if (user?.email) setAccountSummary(user?.email)
+			} else if (loginURL && checkoutURL) {
+				window.location.href = `${loginURL}?redirect=${checkoutURL}`;
```

Comment:
Hi @edwardcho1231 Similar to the previous behavior, the user should be redirected to the login page if he/she is not logged In. However, currently I'm seeing this 

![Screenshot 2024-04-01 at 7 44 44 a m](https://github.com/WPMedia/arc-themes-blocks/assets/54566275/16149f53-0587-4aaa-a819-26bf81a6cc6e)